gapic/api/artman_iam.yaml

```diff
@@ -0,0 +1,26 @@
+common:
+  api_name: iam-v1
```

Comment:
you need a 'google-' prefix on this. Since 'google-iam-v1' already exists (for the mixin), I recommend 'google-iam-admin-v1'.